component/src/main/java/io/siddhi/extension/store/mongodb/MongoDBEventTable.java

```diff
@@ -140,6 +140,10 @@
                                 "the connection with TLS/SSL. false: Initiate the connection without TLS/SSL.",
                         defaultValue = "false",
                         possibleParameters = {"true", "false"}),
+                @SystemParameter(name = "encryptionProtocol",
+                        description = "Encryption protocol when `sslEnabled`.",
```

Comment:
Shall simply just call this "Encryption Protocol"

Reply:
Fixed with https://github.com/siddhi-io/siddhi-store-mongodb/pull/61/commits/be04a98ace2f11b6ba59ad1333899db3d6025ac9